Docs 10981 document console path field feature pa (#367)
mulesoft/docs-api-manager: modules/ROOT/pages/latest-overview-concept.adoc

For details about sharing assets via a private Exchange or an Exchange public portal, see xref:exchange::about-sharing-assets.adoc[Share Assets].

> === API Console

Reviewer: Similarly to task step nesting, it is a good idea to try to limit topic nesting.
While three levels in topic nesting is more acceptable than three in task steps, I looked at this topic title without any context and wondered "What *about* "API Console? What type of information should I expect here?"
Obviously, the superior titles would provide context, so I looked at the unchanged texts and found this:

== API Manager Components and Concepts
=== Policies

Any time that you find yourself with an empty topic level, that has no narrative, ask yourself how you might promote the topics subordinate to that topic. Something to think about going forward.

> When creating or editing APIs in API Manager, you can access the xref:apikit::apikit-simulate[API Console] to view and simulate your API specification. This feature is available for RAML or OAS APIs with Mule v4.x, when you manage the API as an endpoint with proxy.

Reviewer 2: Just a small comment, the _API Console_ feature is also enabled for Mule 3, but it is not possible to disable or modify the console path from API Manager side (if you want to configure it, you should download and modify the proxy manually). 

Before this new feature (_API Console Configuration_), the same happend for Mule 4, but now it is possible to disable or edit the path from API Manager UI (or API).

Author: Done! Thanks!

> When creating or editing APIs in API Manager, you can use the xref:apikit::apikit-simulate[API Console] to view and simulate your API specification. This feature is available for RAML and OAS APIs, when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.

Reviewer: Simply to create a more active voice...:
```suggestion
When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to view and simulate your API specification. This feature is available for RAML and OAS APIs, when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.
```
Does "this feature" refer to the console or to the ability to view and simulate?

Author: Done! Thanks!

> The API Console access from API Manager is disabled by default. However, you can enable the access also modify the API Console path. Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it. The console flow is already included in the API specification when you download an out-of-the-box proxy for your Mule application.

Reviewer 3: ```suggestion
The API Console access from API Manager is disabled by default. However, you can enable the access and also modify the API Console path. Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it. The console flow is already included in the API specification when you download an out-of-the-box proxy for your Mule application.
```

Reviewer 3: Also, this part:
```
Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it
```
Is kind of redundant, considering the paragraph starts mentioning that the console is disabled by default.

Author: @NachoRaik Are completely removing the flow from the proxy XML file and disabling the feature the same thing?

Reviewer 3: Yes, when disabled API Manager takes care of removing the flow from the XML. Right, @tutecano1995 ?

Reviewer 2: Yep, as @NachoRaik mentioned, disabling the API Console from the UI and removing the flow from the XML is equivalent. 

With API Console Configuration, when you download the proxy:
- If API Console  is disabled you will not see the flow in the XML
- If the console path is modified you will see the new console path in the flow
- If the console path is the default (`/console/*`) you will see the default value in the flow

The behavior before this new feature was always having the API Console enabled with `/console/*`

Author: Ok, great thanks! I understood the rest but the part where disabling from the UI means that the flow is removed was not clear. I'll make the change accordingly. Thanks!

Author: Done! Thanks!

> The API Console access from API Manager is disabled by default. However, you can enable the access also modify the API Console path. Optionally, you can entirely remove the console flow from the proxy XML file if you don't need it. The console flow is already included in the API specification when you download an out-of-the-box proxy for your Mule application.

Reviewer 2: Related to the default values for API Console:
- As I mentioned above, for Mule 3 the default behavior is still enabled with `/console/*`
- For Mule 4, from the UI the default behavior will be disabled, but from API the default behavior is still enabled with `/console/*` (so we don't break backwards compatibility from the UI)

Maybe we can rephrase this first sentence, mentioning that this is the case for Mule 4.
```
The API Console access from API Manager is disabled by default.
```

Author: Done! Thanks!

> Starting with Mule 4, you can enable access to the API Console and modify the API Console URL from API Manager. The API Console access from API Manager is disabled by default. For information about how to enable the API Console access fromm API Manager, see xref:manage-exchange-api-task.adoc[Creating an API Instance by Importing From Exchange].

Reviewer: GLOBAL: because API Console is a name, do not precede it with "the" unless also following it with a common noun object.
```suggestion
Starting with Mule 4, you can enable access to API Console and modify the API Console URL from API Manager. API Console access from API Manager is disabled by default. For information about how to enable API Console access from API Manager, see xref:manage-exchange-api-task.adoc[Creating an API Instance by Importing from Exchange].
```

Author: Done! Thanks!

For details about sharing assets via a private Exchange or an Exchange public portal, see xref:exchange::about-sharing-assets.adoc[Share Assets].

=== API Console

> When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to view and test your API specification. You can test RAML and OAS APIs when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.

Reviewer 2: Same comment as above, maybe we could say _"expose and test"_.

Author: Done! Thanks!

> When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to expose and test your API specification. You can test RAML and OAS APIs when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.

Reviewer 4: "xref:apikit::apikit-simulate[API Console]"
Should this be "xref:apikit::apikit-simulate.adoc[API Console]"?

Author: Great catch! Done! Thanks!

> When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to view and simulate your API specification. You can simulate RAML and OAS APIs, when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.

Reviewer: ```suggestion
When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to view and simulate your API specification. You can simulate RAML and OAS APIs when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.
```

Author: Done! Thanks!

> Starting with Mule 4, you can enable access to API Console and modify the API Console URL from API Manager. API Console access from API Manager is disabled by default. For information about how to enable API Console access from API Manager, see xref:manage-exchange-api-task.adoc[Creating an API Instance by Importing from Exchange].

Reviewer 2: Instead of _API Console URL_, I would say _API Console Path_

Author: Done! Thanks!
